### ISSUE TITLE
ReferenceError: err is not defined

### ISSUE THREAD

**Issue by mikelobam** [BeepIsla/csgo-commend-bot#445]
How can I create an err reference for this part.

`targetAcc = (await helper.parseSteamID(config.target)).accountid;`

this is an old version code continuation as follows.

`if (config.method.toUpperCase() === "LOGIN") {
		console.log("white", "Getting an available server");
		serverToUse = (await helper.GetActiveServer()).shift().steamid;

		console.log("white", "Logging into target account");
		targetAcc = new Target(config.account.username, config.account.password, config.account.sharedSecret);
		await targetAcc.login();
	} else if (config.method.toUpperCase() === "SERVER") {
		console.log("white", "Parsing target account...");
		targetAcc = (await helper.parseSteamID(config.target)).accountid;

	}`

all I want to do is create the err reference like in the new version and I want to do it like this;

New Version Code ; 

`else {
		console.log("white", "Parsing target account...");
		sid = await helper.parseSteamID(config.target).catch(async (err) => {
			switch (err.message) {
				case "Failed to parse SteamID": {
					console.log("red", "Input is not in a valid format");
					break;
				}
				case "Invalid Vanity URL": {
					console.log("red", "Input is not a valid SteamID or VanityURL");
					break;
				}
				default: {
					console.error(err);
				}
			}

			await db.close();

			// Force exit the process if it doesn't happen automatically within 15 seconds
			setTimeout(process.exit, 15000, 1).unref();
		});
		if (!sid) {
			return;
		}
	}`

In short, I want to reference err and print the error codes on the screen.

**Comment by BeepIsla** [comment]
targetAcc = await helper.parseSteamID(config.target).catch((err) => {
        // Do whatever you want with "err" here
        console.error(err);
    });
    if (!targetAcc) {
        return;
    }

    targetAcc = targetAcc.accountid;

**Comment by mikelobam** [comment]
thanks this will do my job too much.

**Comment by mikelobam** [comment]
> ```
> targetAcc = await helper.parseSteamID(config.target).catch((err) => {
>     // Do whatever you want with "err" here
>     console.error(err);
> });
> if (!targetAcc) {
>     return;
> }
> 
> targetAcc = targetAcc.accountid;
> ```

it worked but now it gives an error like this;;

TypeError: helper.parseServerID is not a function

**Comment by mikelobam** [comment]
ok i found the problem i forgot to take the helper.js reference, thanks mate.
